integrations/cloudflare.mdx

```diff
@@ -11,150 +11,164 @@ First, enable the Cloudflare integration in the Statsig Console.
+2. Initializing the Statsig client
 3. Checking a Gate
-4. Flushing events to statsig
+4. Logging an event
+5. Flushing events to Statsig
 
-If you've used a statsig sdk in the past, step 2 should be familiar, though you'll be telling the sdk to initialize from the KV store instead of from the statsig backend.
+If you've used a Statsig sdk in the past, these steps should be familiar. The usage will be the same, the only difference is the sdk will initialize from the KV store instead of the statsig backend.
 
-In our example, we are checking a gate called "test_cloudflare_sync" that is set to pass for 50% of everyone.  We create a random userID on every request, and we should see it evaluate to true 50% of the time.
+In our example, we are checking a gate called "test_cloudflare_sync" that is set to a 50% pass rate.  We create a random userID on every request, and we should see it evaluate to true 50% of the time.
 
-### 1. The `CloudflareKVDataAdapter`
-```
-const dataAdapter = new CloudflareKVDataAdapter(env.STATSIG_KV, 'statsig-YOUR_COMPANY_ID');
+### 1. Creating a `StatsigCloudflareClient` instance
+```bash
+const client = new StatsigCloudflareClient("<Your Statsig client key>", {loggingEnabled: 'always'});
 ```
+The client instantiation takes two arguments:
+- `client key : string`  This is available from the [Project Settings](https://console.statsig.com/api_keys) page in the Statsig Console.  This is used to authenticate your requests.  In this example, we have hardcoded it (In prod, set this in the cloudflare dashboard under Worker > Settings > Variables and Secrets)
+- `options : StatsigOptions` If you plan to log events, set `{loggingEnabled: 'always'}` like in the example provided. See here, for more [options](https://docs.statsig.com/client/javascript-sdk#statsig-options).
 
-The adapter takes two arguments:
-- The KV namespace you set up previously.  This must be accessible and [configured as a binding for the worker](https://developers.cloudflare.com/kv/concepts/kv-bindings/)
-- Your statsig project id.  This is the string that comes after `statsig-` in the KV namespace prefix.  You can also find it in any url when you are on the Statsig Console.  For example, `https://console.statsig.com/7LhuarZImmfNdtl9IsDJeX/gates/test_cloudflare_sync/diagnostics` has a project ID of `7LhuarZImmfNdtl9IsDJeX`
+### 2. Client initialization
 
+The following line initializes the client by loading feature gate and experiment configurations directly from your Cloudflare KV store.
 
-### 2. SDK Initialization
-```
-const res = await statsig.initialize(
-    env.STATSIG_SECRET_KEY,
-    { 
-      dataAdapter: dataAdapter,
-      postLogsRetryLimit: 0,
-      initStrategyForIDLists: 'none',
-      initStrategyForIP3Country: 'none',
-      disableIdListsSync: true,
-      disableRulesetsSync: true,
-    },
-);
+```bash
+const initResult = await client.initializeFromCloudflare(env.<YOUR_KV_NAMESPACE_BINDING>, <YOUR_KV_KEY>);
```

Comment:
since we are in the package `@statsig/serverless/cloudflare, should this just be `initialize`?